src/components/Comments.js

```diff
@@ -28,9 +30,21 @@ export const Comments = ({id}) => {
         setIsLoading(true)
         })
         .catch((err)=>{
-            console.log(err)
         })
     }
+
+    if(removeComment){
+        instance.delete(`/comments/${commentId}`).then(()=>{
+        }).catch((err)=>{
+        })
+    }
+
+
+    const deleteComment = ({comment}) => {
+        setCommentId(comment.comment_id)
+        setRemoveComment(true)
+    }
+
```

Comment:
Hi Kelvin, this logic means that if you try to delete multiple comments, you get strange behaviour, where the API call is happening, but the state is updated strangely.